buddyboss/buddyboss-platform
# fix: Unread message count shows wrong at the top bar. #2097

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2097 .

### How to test the changes in this Pull Request:

1. Send a message to Jessica
2. Login as Jessica then go to the messages page.
3. Then click on the (x) icon for the newly appeared thread without viewing the messages.
4. Now you can see the **unread message count** and unread message count from the **message notification** for the hidden threads are marked as read.

### Proof Screenshots or Video
https://vimeo.com/591572578/3bec52652e


### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
